rocky/rocky/scheduler.py

```diff
@@ -123,6 +124,8 @@ class PaginatedTasksResponse(BaseModel):
 
 
 class LazyTaskList:
+    HARD_LIMIT = 99_999_999
```

Comment:
In the context of Rocky, we are never going to need this many tasks, 500 is more than enough